flair/datasets/document_classification.py

```diff
@@ -1541,6 +1541,73 @@ def __init__(self,
         )
 
 
+class GERMEVAL_2018_OFFENSIVE_LANGUAGE(ClassificationCorpus):
+    """
+    GermEval 2018 Task 1 corpus for identification of offensive language.
+    Classifying German tweets into 2 coarse-grained categories OFFENSIVE or OTHER.
+    """
+
+    def __init__(self,
+                 base_path: Union[str, Path] = None,
+                 memory_mode: str = 'full',
+                 tokenizer: Union[bool, Callable[[str], List[Token]], Tokenizer] = SpaceTokenizer(),
```

Comment:
Perhaps change the default tokenizer to `SegtokTokenizer`? It seems the tweets are not tokenized, so the whitespace tokenizer will create some words that include punctuation.

Reply:
Done! Having SegtokTokenizer as the default makes more sense for tweets.